config/filter.d/dovecot.conf

```diff
@@ -14,6 +14,7 @@ failregex = ^%(__prefix_line)s(?:%(__pam_auth)s(?:\(dovecot:auth\))?:)?\s+authen
             ^%(__prefix_line)s(?:Info|dovecot: auth\(default\)|auth-worker\(\d+\)): pam\(\S+,<HOST>\): pam_authenticate\(\) failed: (User not known to the underlying authentication module: \d+ Time\(s\)|Authentication failure \(password mismatch\?\))\s*$
             ^%(__prefix_line)s(?:auth|auth-worker\(\d+\)): (?:pam|passwd-file)\(\S+,<HOST>\): unknown user\s*$
             ^%(__prefix_line)s(?:auth|auth-worker\(\d+\)): Info: ldap\(\S*,<HOST>,\S*\): invalid credentials\s*$
+            ^%(__prefix_line)sdovecot\[\d+]: (imap|pop3-login): Disconnected \(no auth attempts in \d secs\): user=<>, rip=<HOST>, lip=(\d{1,3}\.){3}\d{1,3}. TLS handshaking: (Disconnected|SSL_accept\(\) failed: Unknown error), session=<\S+>?\s*$
```

Comment:
If you can't imagine that it would be a work-case, that doesn't mean it is not really one. For example it could be just a slow connection, etc.
Fail2ban normally not used to prevent DDOS, and firstly it should avoid authentication failures. But secondly it should not ban perfect legal user, that doing something "unexpected" once (e. g. have a slow connection).
You can configure your fail2ban-instance how you want, but for many user of fail2ban it would be possibly unacceptable.
In case of many such "failures" (without auth attempts), it should be another jail with larger `maxretry` or smaller `findtime` (e. g. dovecot-ddos or dovecot-aggressive combined both rules-set).
So IMHO it has nothing to do with normally dovecot-jail.